app/src/preHome.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import Button from './components/Button/Button';
+import FormInput from "./components/FormInput/FormInput";
+import { BrowserRouter as Router, useHistory } from "react-router-dom";
+import { API, Auth } from "aws-amplify";
+import s from "./preHome.module.css";
+
+
+export default function PreHome() {
+    const [isLoading, setIsLoading] = useState(false);
+    const [searchVal, setSearchVal] = useState("");
+    const history = useHistory();
+
+    const toRecordingPage = () => history.push("/recording");
+
+    const handleSubmit = event => {
+        event.preventDefault();
+        alert(`Submitting Name ${searchVal}`)
+        return listSessions()
+    }
+
+    async function listSessions() {
+        const apiName = 'MTADemoAPI';
+        const path = 'listSessions';
+        const myInit = { 
+        //   headers: { 
+        //     Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}`,
+        //   },
+          response: true,
+          queryStringParameters: {
+            PatientId: 'p-3e3477c37d674ecc98e3cdf5487ee07b',
+          },
+        };
+    
+        const result =  await API.get(apiName, path, myInit); 
+        console.log(result);
+        return result;
+    }
+
+    async function listPatients() {
+        const apiName = 'MTADemoAPI';
+        const path = 'listPatients';
+        const myInit = { 
+        //   headers: { 
+        //     Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}`,
+        //   },
+          response: true,
+          queryStringParameters: {
+            PatientId: 'p-3e3477c37d674ecc98e3cdf5487ee07b',
+          },
+        };
+    
+        const result =  await API.get(apiName, path, myInit); 
+        console.log(result);
+        return result;
+    }
+
+    async function listHealthCareProfessionals() {
+        const apiName = 'MTADemoAPI';
+        const path = 'listHealthCareProfessionals';
+        const myInit = { 
+        //   headers: { 
+        //     Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}`,
+        //   },
+          response: true,
+          queryStringParameters: {
+            HealthCareProfessionalId: 'h-67a2c81c20994ff48b52c1b7416667a5',
```

Comment:
Please update the functions to take in arguments and replace the hardcoded variables